src/core/internal/dassert.d

```diff
@@ -37,21 +37,21 @@ private template getPrintfFormat(T)
 Minimalistic formatting for use in _d_assert_fail to keep the compilation
 overhead small and avoid the use of Phobos.
 */
-auto miniFormat(V)(auto ref V v)
+auto miniFormat(V)(V v)
 {
     import core.stdc.stdio : sprintf;
     import core.stdc.string : strlen;
     static if (__traits(isIntegral, V))
     {
         enum printfFormat = getPrintfFormat!V;
         char[20] val;
-        sprintf(val.ptr, printfFormat, v);
+        sprintf(&val[0], printfFormat, v);
         return val.idup[0 .. strlen(val.ptr)];
```

Comment:
`val.ptr` -> `&val[0]`

Reply:
This has to be `@trusted` code anyhow :/